lib/src/analytics.dart

```diff
@@ -26,6 +31,10 @@ abstract class Analytics {
         branch: branch,
         flutterVersion: flutterVersion,
         dartVersion: dartVersion,
+        platform: platform,
+        toolsMessage: kToolsMessage,
+        toolsMessageVersion: kToolsMessageVersion,
+        fs: LocalFileSystem(),
```

Comment:
I think this can/should be const
```suggestion
        fs: const LocalFileSystem(),
```